content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/While/1. While.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 1. While
+taxonomie:
+  - ib-19.2.Controlestructuur-Herhaling-While
+  - ib-19.3.Controlestructuur-Herhaling-While
+draft: false
+---
+
+## Wat is een While-loop?
```

Comment:
Misschien een idee om in een tweede bestand dingen zoals `break` en `continue` ook uit te leggen.
https://www.w3schools.com/cs/cs_break.php

Reply:
dat is een hele andere taxonomie code